webapp/src/main/resources/locale/portlet/matrix_en.properties

```diff
@@ -26,3 +26,5 @@ matrix.chat.audio.pause.label=Pause
 matrix.message.edited.label=edited
 matrix.chat.message.deleted=Message deleted
 matrix.chat.message.label=Message
+matrix.chat.upload.image.ignored.message={0} image(s) could not be uploaded because they exceed the maximum allowed size
```

Comment:
could you add the max file size allowed here so that end user understands the issue